donkeycar/utils.py

```diff
@@ -428,15 +428,17 @@ def eprint(*args, **kwargs):
     print(*args, file=sys.stderr, **kwargs)
 
 
-def get_model_by_type(model_type: str, cfg: 'Config') -> 'KerasPilot':
+def get_model_by_type(model_type: str, cfg: 'Config') -> 'ABC':
```

Comment:
You can return `Union['KerasPilot', 'FastAiPilot']`

Reply:
Done